doc/whats_new.rst

```diff
@@ -22,6 +22,10 @@ Current
 Changelog
 ~~~~~~~~~
 
+- Add existence check for :func:`write_raw_bids` before :func:`make_dataset_description` is called, by `Adam Li`_ (`#331 <https://github.com/mne-tools/mne-bids/pull/331>`_)
```

Comment:
normally we add the new changes at the bottom not at the top, so the whats_new file reads chronologically and not reverse-chronologically